app/src/main/java/org/systers/mentorship/models/HomeStatistics.kt

```diff
@@ -0,0 +1,39 @@
+package org.systers.mentorship.models
+
+import android.annotation.SuppressLint
+import android.os.Parcelable
+import com.google.gson.annotations.SerializedName
+import kotlinx.android.parcel.Parcelize
+
+/**
+ * This data class represents the response of the /home endpoint
```

Comment:
You don't need to reference the endpoint. You never know if this can be reused for something else.
Say:
```
This class represents statistics of the user actions on the app.
```

Reply:
Okay, fixed now. I've also removed the unnecessary blank line below for consistency.